PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ArtisticEffectFormat.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Microsoft.Office.Core;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Utils;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+using Shapes = Microsoft.Office.Interop.PowerPoint.Shapes;
+
+namespace PowerPointLabs.SyncLab.ObjectFormats
+{
+    class ArtisticEffectFormat: Format
+    {
+        public override bool CanCopy(Shape formatShape)
+        {
+            try
+            {
+                return formatShape.Fill.PictureEffects.Count > 0;
+            } 
+            catch
+            {
+                return false;
+            }
+        }
+
+        public override void SyncFormat(Shape formatShape, Shape newShape)
+        {
+            if (!Sync(formatShape, newShape))
+            {
+                Logger.Log(newShape.Type + " unable to sync ArtisticEffect Format");
+            }
+        }
+
+        public override Bitmap DisplayImage(Shape formatShape)
+        {
+            Bitmap image = GraphicsUtil.ShapeToBitmap(formatShape);
+            return image;
+        }
+
+        public static List<MsoPictureEffectType> GetArtisticEffects(Shape shape)
+        {
+            List<MsoPictureEffectType> artisticEffects = new List<MsoPictureEffectType>();
+            try
+            {
+                PictureEffects effects = shape.Fill.PictureEffects;
```

Comment:
Is it possible to choose multiple artistic effects? How do you do it (I am using ppt 2013). If it is possible, then our tests are a bit lacking, we should have a test for this case.

Reply:
I think you can do so by adding one artistic effect after another. Selecting a new effect adds a new one, rather than replaces the old one. There's a 'clear' button which clears all artistic effects.

Reply:
So are you going to add tests for such a case? Or is it already covered by one of the tests?

Reply:
Deepest apologies Ian I've made a mistake on my statement that we can apply more than 1 artistic effect on picture, 
It appears that we can't actually stack artistic effects despite how it sounds in the API. It does however, make me wonder why the API treats ArtisticEffect as a collection of effects. 
I'll note this down as a comment in the ArtisticEffectFormat file.